src/gui/LocationDialog.cpp

```diff
@@ -975,7 +975,7 @@ void LocationDialog::resetGPSbuttonLabel()
 #else
 	ui->gpsToolButton->setText(q_("Get location from GPS"));
 #endif
-	ui->gpsToolButton->setStyleSheet(QString("QToolButton{ background: gray; }"));
+	ui->gpsToolButton->setStyleSheet("");
```

Comment:
Apparently, this line should be removed

Reply:
While you were right about the creation of dialog content, here the style must be actively reset after having been set to a red or green color.